extract_pov_edits/dataflow_main.py

```diff
@@ -367,4 +316,14 @@ def directory(mirror):
            sections.append(obj.name[obj.name.rfind('/') + 1:])
   if known_args.ingestFrom == 'local':
      sections = [known_args.localStorage]
-  run(known_args, pipeline_args, sections)
+  if known_args.start is not None:
```

Comment:
just:
if know_args.start:

but prob
start = known_args.start or 0
end = known_args.end or 0

Reply:
Yeah that's the kind of confusion I want to avoid here, they can be 0.

Reply:
OK